tests/auth_tests/models/custom_user.py

```diff
@@ -95,6 +97,34 @@ def __exit__(self, exc_type, exc_value, traceback):
         PermissionsMixin._meta.local_many_to_many = self._old_pm_local_m2m
 
 
+class CustomUserWithoutIsActiveField(AbstractBaseUser):
+    username = models.CharField(
+        _('username'),
+        max_length=150,
+        unique=True,
+        help_text=_('Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.'),
+        validators=[
```

Comment:
try to make this as minimal as possible -- if we don't need all the validators, error_messages, etc. leave them out